common-arcana.lic

```diff
@@ -51,6 +59,29 @@ module DRCA
     ['Your body is willing', 'You have not recovered'].include?(result)
   end
 
+  def start_barb_abilities(skills, settings)
+    skills
+      .each do |name|
+        next if DRSpells.active_spells[name]
```

Comment:
Another option would be to `.reject { |name| DRSpells.active_spells[name] }.each do ....`. To me that makes it a bit clearer what each function in the chain is doing.

Note: I'm just pointing out a style difference, you don't have to change this.

Reply:
No, this is what I want from reviews! Thanks a lot!

Reply:
Definitely wanting to use more of what Ruby has to offer, I just always forget about select/reject.